src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+  const communityAccessibilityElement = (
+    <a href={props.communityAccessibilityLink}>Website Accessibility Policy</a>
+  );
+
+  /* eslint-disable max-len */
+  return (
```

Comment:
Does all this text need to be internationalized? Or is that not in scope or relevant?

Reply:
I don't believe it needs to be internationalized. Only the link in the footer to this page needs to be.